components/Header/Header.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import Image from "next/image";
+import Logo from "../../assets/images/Logo(cragList).png";
+
+export default function Header() {
+  return (
+    <>
+      <StyledHeader>
+        <Image src={Logo} width={100} />
+      </StyledHeader>
+    </>
```

Comment:
```suggestion
      <StyledHeader>
        <Image src={Logo} width={100} />
      </StyledHeader>
```
The fragments are only necessary if you have two parent elements inside of your header. :)